lib/class-wp-duotone-gutenberg.php

```diff
@@ -349,7 +349,26 @@ public static function render_duotone_support( $block_content, $block ) {
 		$filter_id = gutenberg_get_duotone_filter_id( array( 'slug' => $slug ) );
 
 		// Build the CSS selectors to which the filter will be applied.
-		$selector = WP_Theme_JSON_Gutenberg::scope_selector( '.' . $filter_id, $duotone_selector );
+		$scopes    = explode( ',', '.' . $filter_id );
```

Comment:
I wish I hadn't to paste this here, but this needs to work differently than the `WP_Theme_JSON_Gutenberg::scope_selector` (do not add an extra space).

Ideas for a follow-up PR:

- The usage of this internal utility has expanded to: [settings](https://github.com/WordPress/gutenberg/blob/70198b9811c1bdc99579e272a72af3dae0ec9138/lib/block-supports/settings.php#L95), [duotone](https://github.com/WordPress/gutenberg/blob/trunk/lib/class-wp-duotone-gutenberg.php#L352), [WP_Theme_JSON class](https://github.com/WordPress/gutenberg/blob/70198b9811c1bdc99579e272a72af3dae0ec9138/lib/class-wp-theme-json-gutenberg.php#L998). It's worth creating a utility function somewhere. Where? `wp_scope_css_selector( $scope, $selector ) ` as a name?

- The consumer should be responsible for adding or not adding spaces, not the function itself:
  - `wp_scope_css_selector( $scope . ' ', $selector )` if the consumer wants spaces.
  - `wp_scope_css_selector( $scope . ' ', $selector )` if the consumer doesn't want spaces.

Thoughts?